contracts/Strategy.sol

```diff
@@ -6,42 +6,29 @@ pragma solidity 0.6.12;
 pragma experimental ABIEncoderV2;
 
 // These are the core Yearn libraries
-import {
-    BaseStrategy,
-    StrategyParams
-} from "@yearnvaults/contracts/BaseStrategy.sol";
-import {
-    SafeERC20,
-    SafeMath,
-    IERC20,
-    Address
-} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import {BaseStrategy} from "@yearnvaults/contracts/BaseStrategy.sol";
+import {SafeERC20, SafeMath, IERC20, Address} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
 import "@openzeppelin/contracts/math/Math.sol";
 
-
 // Import interfaces for many popular DeFi projects, or add your own!
 import "../interfaces/I1INCHGovernance.sol";
 import "../interfaces/I1INCHGovernanceRewards.sol";
-interface I1nchStake is I1INCHGovernance, IERC20 {}
+
 contract Strategy is BaseStrategy {
     using SafeERC20 for IERC20;
     using Address for address;
     using SafeMath for uint256;
 
-    IERC20 public OneInchToken = IERC20(0x111111111117dC0aa78b770fA6A738034120C302);
     //Initiate 1inch interfaces
-    I1nchStake public stakeT = I1nchStake(0xA0446D8804611944F1B527eCD37d7dcbE442caba);
+    I1INCHGovernance public stakeT = I1INCHGovernance(0xA0446D8804611944F1B527eCD37d7dcbE442caba);
     I1INCHGovernanceRewards public governanceT = I1INCHGovernanceRewards(0x0F85A912448279111694F4Ba4F85dC641c54b594);
-    // The amount of 1INCH we have earned which we are now staking in 1INCH Governance
-    uint256 public staking = 0;
 
     constructor(address _vault) public BaseStrategy(_vault) {
-        require(address(want) == address(OneInchToken),"wrong want token");
```

Comment:
it didn't make sense to me to have a variable in storage just to check that the constructor is doing things right. Up to you.